alpaca/rest.go

```diff
@@ -81,8 +84,15 @@ func NewClient(opts ClientOpts) Client {
 	if opts.RetryDelay == 0 {
 		opts.RetryDelay = time.Second
 	}
+	httpClient := opts.HttpClient
+	if httpClient == nil {
+		httpClient = &http.Client{
+			Timeout: opts.Timeout,
```

Comment:
What if the client provides both options: HttpClient and Timeout?

Reply:
Good question. The HttpClient has precedence. I'll add a comment.